library/psa_crypto.c

```diff
@@ -2238,6 +2248,11 @@ psa_status_t psa_mac_sign_finish( psa_mac_operation_t *operation,
 {
     psa_status_t status;
 
+    if( operation->alg == 0)
```

Comment:
Style: missing space before `)` (here and elsewhere)

Reply:
Gah, I thought I fixed all of these...